CHANGELOG.md

```diff
@@ -2,18 +2,15 @@
 
 ## 0.7.6dev
 
+* [Feature] Adds `--persist-replace` argument to replace existing tables when persisting data frames (#440)
+
 ## 0.7.5 (2023-05-24)
 
 * [Feature] Using native DuckDB `.df()` method when using `autopandas`
 * [Feature] Better error messages when function used in plotting API unsupported by DB driver ([#159](https://github.com/ploomber/jupysql/issues/159))
 * [Feature] Detailed error messages when syntax error in SQL query, postgres connection password missing or inaccessible, invalid DuckDB connection string ([#229](https://github.com/ploomber/jupysql/issues/229))
 * [Fix] Fix the default value of %config SqlMagic.displaylimit to 10 ([#462](https://github.com/ploomber/jupysql/issues/462))
 * [Doc] documenting `%sqlcmd tables`/`%sqlcmd columns`
-* [Feature] Better error messages when function used in plotting API unsupported by DB driver (#159)
-* [Fix] Fix the default value of %config SqlMagic.displaylimit to 10 (#462)
-* [Feature] Detailed error messages when syntax error in SQL query, postgres connection password missing or inaccessible, invalid DuckDB connection string (#229)
-* [Feature] Adds `--persist-replace` argument (#440)
-
```

Comment:
looks like some rebase went bad? I see some entries in the changelog are deleted

Reply:
Can you check again? I rebase again
<img width="1524" alt="Screenshot 2023-05-27 at 4 31 35 AM" src="https://github.com/ploomber/jupysql/assets/123580782/6aa11463-e344-4fbf-9906-75fba2b604e5">